### DISCUSSION TITLE
Path error on build?

### DISCUSSION THREAD

**Discussion by djeusette** [vercel/vercel#4931] [Help] [answered]
Hi there!

During build time, I keep getting this error in the logs:

```
00:01:17.135 | Error: File /vercel/5b518e27/node_modules/apollo-upload-clientpublic/ReactNativeFile does not exist.
00:01:17.135 | at Job.emitDependency (/vercel/42f2bbfe8d152ee9/.build-utils/.builder/node_modules/@vercel/next/dist/index.js:9092:23)
00:01:17.135 | at /vercel/42f2bbfe8d152ee9/.build-utils/.builder/node_modules/@vercel/next/dist/index.js:9121:36
00:01:17.135 | at async Promise.all (index 0)
```

Note the path issue: `/vercel/5b518e27/node_modules/apollo-upload-clientpublic/ReactNativeFile`. A slash is definitely missing between `apollo-upload-client` and `public`.

On the library issue (https://github.com/jaydenseric/apollo-upload-client/issues/211), different possible reasons were mentioned:

- https://github.com/vercel/next.js/issues/8113
- https://github.com/vercel/vercel/pull/4814
- https://github.com/vercel/node-file-trace/pull/128

Does anyone have a workaround?

Thanks

**Comment by styfle** [comment, marked as answer]
Hi @djeusette 

This has been fixed on the canary channel and will be available in the next stable release.

You can run `vc -b FORCE_BUILDER_TAG=canary` from the CLI or add the environment variable to the UI to opt-in.

https://vercel.com/blog/environment-variables-ui

**Reply by djeusette** [reply]
Sweet, thank you.
